app/Filament/Resources/DatabaseHostResource/Pages/ListDatabaseHosts.php

```diff
@@ -31,14 +31,17 @@ public function table(Table $table): Table
                 TextColumn::make('port')
                     ->sortable(),
                 TextColumn::make('username')
+                    ->icon('tabler-user')
                     ->searchable(),
-                TextColumn::make('max_databases')
-                    ->numeric()
-                    ->sortable(),
```

Comment:
Why was this removed?

Reply:
"Max" limit for a resource isn't a good metric to display on the overview. You display current metrics instead.